src/build.py

```diff
@@ -586,6 +609,13 @@ def SyncGNUWin32(name, src_dir, git_repo):
   return SyncArchive(GNUWIN32_DIR, name, url)
 
 
+def SyncPrebuiltJava(name, src_dir, git_repo):
+  platform = {'linux2': 'linux', 'darwin': 'osx', 'win32': 'x64'}[sys.platform]
```

Comment:
should win32 be 'windows' instead of 'x64'?

Reply:
😱 Done